ui/src/app/memberships/page.tsx

```diff
@@ -15,12 +26,16 @@ const Memberships = () => {
             </div>
             <TabsContent value="current-memberships">
                 <div className="bg-white">
-                    <div className="container">{/* MembershipsTable goes here */}</div>
+                    <div className="container">
+                        <MembershipsTable results={results} isOptOut={true} />
+                    </div>
                 </div>
             </TabsContent>
             <TabsContent value="membership-opportunities">
                 <div className="bg-white">
-                    <div className="container">{/* MembershipsTable goes here */}</div>
+                    <div className="container">
+                        <MembershipsTable results={groupingPaths} isOptOut={false} />
+                    </div>
```

Comment:
Could you split this page where the tabs "Current Memberships" and "Membership Opportunities" have their own link?

1. "Current Memberships" tab: /memberships and /memberships/current
2. "Membership Opportunities" tab: /memberships/opportunities

You can follow Isaac's code on the Admin page in his PR: https://github.com/uhawaii-system-its-ti-iam/uh-groupings/pull/114/files. Thanks!